docs/en/sql-reference/statements/alter.md

```diff
@@ -201,17 +201,17 @@ All changes on replicated tables are broadcasting to ZooKeeper so will be applie
 
 The following operations with [partitions](../../engines/table-engines/mergetree-family/custom-partitioning-key.md) are available:
 
--   [DETACH PARTITION](#alter_detach-partition) – Moves a partition to the `detached` directory and forget it.
--   [DROP PARTITION](#alter_drop-partition) – Deletes a partition.
--   [ATTACH PART\|PARTITION](#alter_attach-partition) – Adds a part or partition from the `detached` directory to the table.
--   [ATTACH PARTITION FROM](#alter_attach-partition-from) – Copies the data partition from one table to another and adds.
--   [REPLACE PARTITION](#alter_replace-partition) - Copies the data partition from one table to another and replaces.
--   [MOVE PARTITION TO TABLE](#alter_move_to_table-partition)(#alter_move_to_table-partition) - Move the data partition from one table to another.
```

Comment:
I suppose, the anchor was duplicated by mistake or this doubling makes any sense?

Reply:
There is no any sense in such duplication.